### PR TITLE
DOCSP-40106 C2C Beta Disclaimer

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION 
Adds a notice for users that they must accept the disclaimer when they first run the `mongosync` beta binary. 

## STAGING 
https://preview-mongodbajhuhmdb.gatsbyjs.io/cluster-sync/DOCSP-40106-beta-disclaimer/reference/beta-program/#get-started

## JIRA 
https://jira.mongodb.org/browse/DOCSP-40106

## BUILD
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=666b52b1ebce19f314aa11d2

### CODE REVIEW

File: source/reference/beta-program.txt [hunk 1]
```diff
@@ -23,25 +23,36 @@ Get Started
...
+following disclaimer:
+
+.. code-block:: shell
+   :copyable: false
+  
+   MongoDB restricts access to beta features via the beta build. Your use 
+   of the beta is governed by the language specified in the Cloud 
+   Subscription Agreement, Cloud Terms of Service, or other applicable 
+   agreement between you and MongoDB with respect to your use of 
+   ``mongosync``.
```
Comment (kennethdyer) [inline source/reference/beta-program.txt:36]: Question: Does the shell output include the double ticks?  I imagine it shouldn't but we should follow whatever the user sees in the output.

**Review by kennethdyer — APPROVED** [review]
Thanks for this @ajhuh-mdb, LGTM with a question.

File: source/reference/beta-program.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/reference/beta-program.txt:36]: Ah that was left over from when this used to be an admonition instead of a code block. Updating.

**Comment by ajhuh-mdb** [comment]
LGTM'd by Revathy via Slack.

**Pull request merged**